test/distributed.in.cpp

```diff
@@ -31,12 +30,13 @@ using namespace ngraph;
 
 TEST(distributed_${BACKEND_NAME}, allreduce)
 {
+    Distributed dist_instance;
```

Comment:
One comment for the unit test is: set the environment variable NGRAPH_CODEGEN=TRUE to check the code from cpu_emitter. The default only checks the code from builder (IIRC).